src/spring-cloud/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+3.1.5
+---
+* The argument '--build-env' accepts key[=value] instead of json.
```

Comment:
It is still recommended to mark it as breaking change, even if its impact is small, because this is our specification for breaking change
```suggestion
* [BREAKING CHANGE] The argument '--build-env' accepts key[=value] instead of json.
```